README.md

```diff
@@ -235,6 +236,120 @@ If you need the latest code from upstream git of OpenSCAP and/or
 SCAP Security Guide instead, pass `--openscap-from-git` and/or
 `--ssg-from-git` to the `./generate-dockerfile.py`.
 
+## REST API
+
+
+### REST API Endpoints
+
+| Endpoint                                  | VERB   | CLI Equivalent                               | Description                                               |
+|-------------------------------------------|--------|----------------------------------------------|-----------------------------------------------------------|
+| /tasks/                                    | GET    | oscapd-cli task                              | Gets all tasks with its information                       |
+| /tasks/                                    | POST   | oscapd-cli task-create -i                    | Creates new tasks                                         |
+| /tasks/<int:taskId>/                       | PUT    | oscapd-cli task <taskId> set-XXX             | Modify tasks                                              |
+| /tasks/<int:taskId>/                       | GET    | oscapd-cli task <taskId>                     | Gets information about <taskId>                           |
+| /tasks/<int:taskId>/                       | DELETE | oscapd-cli task <taskId> remove              | Remove task <taskId>                      |
+| /tasks/<int:taskId>/results/                      | DELETE | oscapd-cli task <taskId> remove              | Remove task <taskId> and its results                      |
```

Comment:
If this is here, does it mean the previous interface removes the task and keeps the results?

Reply:
No. You cannot remove a task which has results. You should remove first all its results before being able to remove the task itself. The second endpoint will first remove all the results and then remove the task.

Reply:
Thanks for explanation. Then I doubt that then first endpoint is useful, because it seems that it can be fully replaced by using the second one.

Reply:
Both endpoints were implemented in order to "simulate" the behaviour on "oscapd-cli" where you have the following options:

oscapd-cli task <id> remove 
oscapd-cli task <id> remove --remove-results

If you think this is not needed here I'm okay deleting "-X DELETE /tasks/<int:taskId>/" endpoint.

Reply:
I don't have a strong opinion on this, I just can't imagine any use-case for the first endpoint, but I might be missing something. If you also don't consider that endpoint useful, then remove it.

Reply:
It's mostly for sanity when writing/implementing automation. If your code assumes there are no results for that tasks you might as well call the first end-point to verify that assumption

Reply:
Based on @mpreisler response, I will keep the endpoint present.